src/templates/luneClient.hbs

```diff
@@ -21,14 +21,24 @@ export class LuneClient {
     protected client: AxiosInstance
     protected config: ClientConfig
 
-    constructor(apiKey: string, baseUrl: string = 'https://api.lune.co', apiVersion: string = '1') {
+    constructor(
+        apiKey: string,
+        baseUrl: string = 'https://api.lune.co',
+        apiVersion: string = '1',
+        account: string,
```

Comment:
I don't think we can have required parameters after optional/default-valued parameters

Reply:
Ups, this should be optional. Fixing 🙈